lib/rspec/sidekiq/matchers/base.rb

```diff
@@ -118,8 +118,19 @@ def initialize(klass)
           @jobs = unwrap_jobs(klass.jobs).map { |job| EnqueuedJob.new(job) }
         end
 
-        def includes?(arguments, options)
-          !!jobs.find { |job| matches?(job, arguments, options) }
+        def includes?(arguments, options, count)
+          matching = jobs.filter { |job| matches?(job, arguments, options) }
+
+          case count
+          in [:exactly, n]
+            matching.size == n
+          in [:at_least, n]
+            matching.size >= n
+          in [:at_most, n]
```

Comment:
💭 (just a musing I wrote out, no action or question here!) At first I was kinda unconvinced about `@expected_count` being an array form. I was leaning towards recommending 2 variables or passing in some sort of object/struct containing the pieces, but this `case` statement form has grown on me. Unexpectedly readable and discernible!

Reply:
Same here 🙂 I started with the simplest implementation possible, but ended up liking it.